### PR TITLE
3rd party repo: add h2 v1.3.176

### PR DESCRIPTION
**Pull request by maggu2810**
This PR could be merged as soon as the CQ for "H2 Database Engine Version: 1.3.176" has been approved.

### CODE REVIEW

File: third-party/pom.xml [hunk 1]
```diff
@@ -69,6 +69,15 @@
                   <id>tec.units:unit-ri:0.8</id>
                   <transitive>false</transitive>
                 </artifact>
+                <artifact>
+                  <!-- https://dev.eclipse.org/ipzilla/show_bug.cgi?id=11900 -->
+                  <id>com.h2database:h2:1.3.176</id>
+                  <transitive>false</transitive>
+                </artifact>
+                <artifact>
+                  <id>org.osgi:org.osgi.service.jdbc:jar:1.0.0</id>
```
Comment (kaikreuzer) [inline third-party/pom.xml]: This file is not part of CQ 11900, so I guess I cannot merge this PR.
Or did I miss a second CQ?

Comment (maggu2810) [inline third-party/pom.xml]: You are right, this file is not part of the CQ 11900.
Do we need a CQ for the OSGi specification package, too?
If so, I will create one for.

Comment (kaikreuzer) [inline third-party/pom.xml]: Yes, we need CQs for ANY third party code...

Comment (maggu2810) [inline third-party/pom.xml]: approved

**Pull request merged**